### PR TITLE
use go native zstd

### PR DESCRIPTION
**Pull request by pschou**

### CODE REVIEW

File: deb/ar_test.go [hunk 1]
```diff
@@ -7,7 +7,7 @@ import (
 	"os"
 	"testing"
 
-	"pault.ag/go/debian/deb"
+	"github.com/pschou/go_debian/deb"
```
Comment (paultag) [inline deb/ar_test.go:10]: URL path here is still your test repo.

File: deb/ar.go [hunk 2]
```diff
@@ -18,7 +18,7 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE. }}} */
 
-package deb // import "pault.ag/go/debian/deb"
```
Comment (paultag) [inline deb/ar.go:21]: Please retain these, if we need to remove them, let's start a new PR removing them repo-wide. I'm guessing this is due to the package replacement that's still intact below

**Review by paultag — CHANGES_REQUESTED** [review]

File: deb/ar.go (hunk 2 above)
Comment (pschou) [reply deb/ar.go:21]: Ok--

Pro: if you remove them, then anyone can host the same package elsewhere and not have GO throw alerts that it isn't in the right place
Con: the control of the sourcecode baseline can help with keeping the changes centralized.

Comment (paultag) [reply deb/ar.go:21]: Yeah totally. We can discuss the merits of this one in another PR, this style is more common in pre go.mod code so I'm not opposed on it's face. Let's keep that change out of this PR if you don't mind though!

**Review by paultag — APPROVED** [review]

**Comment by paultag** [comment]
Thank you!!

**Pull request merged**